src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -297,9 +297,8 @@ export class CourseUpdateComponent implements OnInit {
             course['courseInformationSharingConfiguration'] = CourseInformationSharingConfiguration.COMMUNICATION_AND_MESSAGING;
         } else if (this.communicationEnabled && !this.messagingEnabled) {
             course['courseInformationSharingConfiguration'] = CourseInformationSharingConfiguration.COMMUNICATION_ONLY;
-        } else if (!this.communicationEnabled && this.messagingEnabled) {
-            course['courseInformationSharingConfiguration'] = CourseInformationSharingConfiguration.MESSAGING_ONLY;
```

Comment:
This makes `MESSAGING_ONLY` an impossible configuration.

We should make sure on the server that setting this option is no longer possible and maybe even remove this enum value + write a small migration.